tests/Command/CloneSiteCommandTest.php

```diff
@@ -63,6 +60,9 @@ protected function setUp(): void
         $this->application->add($command);
     }
 
+    /**
+     * @group legacy
```

Comment:
why group legacy?

Reply:
Answer myself to the question, because this is needed: https://github.com/sonata-project/SonataPageBundle/pull/1488
We should remember to remove this on that PR.